From 873174df29160dd8e7a3ee8b2b261aed5e688e0b Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 12 Nov 2021 13:20:27 -0800
Subject: [PATCH] Python 3.10 compat

---
 cMake/FreeCAD_Helpers/SetupPython.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cMake/FreeCAD_Helpers/SetupPython.cmake b/cMake/FreeCAD_Helpers/SetupPython.cmake
index 3c8e106..f5ffb9a 100644
--- a/cMake/FreeCAD_Helpers/SetupPython.cmake
+++ b/cMake/FreeCAD_Helpers/SetupPython.cmake
@@ -3,7 +3,7 @@ macro(SetupPython)
 
     # http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=677598
     # Acceptable versions of Python
-    set(Python_ADDITIONAL_VERSIONS "2.7")
+    set(Python_ADDITIONAL_VERSIONS "3.10")
 
     # For building on OS X
     if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin" AND NOT BUILD_WITH_CONDA)
-- 
2.33.0

