--- FreeCAD-0.17/src/Mod/Path/libarea/CMakeLists.txt~	2018-04-06 08:48:37.000000000 +0000
+++ FreeCAD-0.17/src/Mod/Path/libarea/CMakeLists.txt	2018-12-21 17:15:40.678186908 +0000
@@ -17,7 +17,7 @@
 if(USE_BOOST_PYTHON)
     if(NOT FREECAD_LIBPACK_USE OR FREECAD_LIBPACK_CHECKFILE_CLBUNDLER)
         if(NOT PYTHON_VERSION_MAJOR LESS 3)
-            find_package( Boost COMPONENTS python3)
+            find_package( Boost COMPONENTS python37)
             if (NOT Boost_PYTHON3_FOUND)
                 find_package( Boost COMPONENTS python REQUIRED)
             endif()
--- FreeCAD-0.17/src/Mod/Path/libarea/CMakeLists.txt~	2018-12-21 17:15:40.000000000 +0000
+++ FreeCAD-0.17/src/Mod/Path/libarea/CMakeLists.txt	2018-12-21 17:23:26.176180427 +0000
@@ -19,7 +19,7 @@
         if(NOT PYTHON_VERSION_MAJOR LESS 3)
             find_package( Boost COMPONENTS python)
             if (NOT Boost_PYTHON3_FOUND)
-                find_package( Boost COMPONENTS python REQUIRED)
+                find_package( Boost COMPONENTS python37 REQUIRED)
             endif()
         else()
             find_package( Boost COMPONENTS python REQUIRED)  # find BOOST and boost-python
