--- FreeCAD-0.17/src/CXX/Python3/IndirectPythonInterface.hxx~	2018-04-06 08:48:37.000000000 +0000
+++ FreeCAD-0.17/src/CXX/Python3/IndirectPythonInterface.hxx	2018-12-21 17:30:58.531174129 +0000
@@ -178,7 +178,7 @@
 PYCXX_EXPORT void _XINCREF( PyObject *op );
 PYCXX_EXPORT void _XDECREF( PyObject *op );
 
-PYCXX_EXPORT char *__Py_PackageContext();
+PYCXX_EXPORT const char *__Py_PackageContext();
 };
 
 #endif    // __CXX_INDIRECT_PYTHON_INTERFACE__HXX__
--- FreeCAD-0.17/src/CXX/IndirectPythonInterface.cxx~	2018-04-06 08:48:37.000000000 +0000
+++ FreeCAD-0.17/src/CXX/IndirectPythonInterface.cxx	2018-12-21 17:27:58.988176629 +0000
@@ -549,7 +549,7 @@
 PYCXX_EXPORT int &_Py_OptimizeFlag()                 { return Py_OptimizeFlag; }
 PYCXX_EXPORT int &_Py_NoSiteFlag()                   { return Py_NoSiteFlag; }
 PYCXX_EXPORT int &_Py_VerboseFlag()                  { return Py_VerboseFlag; }
-PYCXX_EXPORT char *__Py_PackageContext()             { return _Py_PackageContext; }
+PYCXX_EXPORT const char *__Py_PackageContext()             { return _Py_PackageContext; }
 
 //
 //    Needed to keep the abstactions for delayload interface
